<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM_ControlPort2</class>
 <widget class="QWidget" name="FORM_ControlPort2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(:/pic_de260/bg_01.png);*/
QLabel{
color: rgb(15, 175, 252);
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="portinfo">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>主控制器DI端口查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pageinfo">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>第2/6页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>控制器I/O</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/ico_260/915.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_5">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text6">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>411</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI6-右转向信号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text3">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>288</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI3-空档</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_2">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>288</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text4">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>329</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI4-前进档信号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_4">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>329</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text5">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>370</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI5-左转向信号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>247</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI2-倒车档</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_3">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>247</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>206</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>299</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI1-驻车档            </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_1">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>206</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="light_6">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>411</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text7">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>452</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI7-车梯灯</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_7">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>452</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text9">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>493</y>
     <width>299</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>雨刮间歇信号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_8">
   <property name="geometry">
    <rect>
     <x>1118</x>
     <y>493</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text2_2">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>206</y>
     <width>311</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI12-空滤器堵塞报警1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_12">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>206</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="light_14">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>288</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text5_2">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>329</y>
     <width>311</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI15-空滤器堵塞报警4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_15">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>329</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text3_2">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>247</y>
     <width>311</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI13-空滤器堵塞报警2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_13">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>247</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text4_2">
   <property name="geometry">
    <rect>
     <x>1182</x>
     <y>288</y>
     <width>311</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI14-空滤器堵塞报警3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
